source/_integrations/nest.markdown

```diff
@@ -299,6 +299,31 @@ All Google Nest Cam models and the Google Nest Hello Video Doorbell support [Dev
 
 See [Automating Home Assistant](/getting-started/automation/) for the getting started guide on automations or the [Automation](/docs/automation/) documentation for full details.
 
+### Example
+
+This automation will trigger when a `nest_event` event type with a type of `camera_motion` is received from the specified `device_id`.
+
+```yaml
+alias: motion alert
+description: ''
```

Comment:
Let's either add an actual description or remove the option.

Reply:
:+1: Removed